src/client/datascience/interactive-common/notebookUsageTracker.ts

```diff
@@ -17,7 +17,7 @@ import { INotebookEditor, INotebookEditorProvider } from '../types';
 export class NotebookUsageTracker implements IExtensionSingleActivationService {
     protected readonly _onDidChangeActiveNotebookEditor = new EventEmitter<INotebookEditor | undefined>();
     protected readonly _onDidOpenNotebookEditor = new EventEmitter<INotebookEditor>();
-    private readonly executedEditors = new Set<INotebookEditor>();
+    private readonly executedNotebooksIndexedByUri = new Set<string>();
```

Comment:
Removed reliance on INotbeookEditor, we don't really need that.
Using the new VS Code API.